client/src/components/molecules/ClientRequestTable.tsx

```diff
@@ -0,0 +1,154 @@
+        softDeleteRequest(id: $id) {
+            id
+            success
+            message
+        }
+    }`
+    
+    useEffect(() => {
+        const undeletedReq : Request[] = props.requests.filter((request)=> request.deleted === false);
+        const unfulfilledReq = undeletedReq.filter(request => {
+            if (request !== undefined){
+                if (request.fulfilled === false){
+                    return request;
+                }
+            }
+        });
+        const fulfilledRequests = undeletedReq.filter(request => {
+            if (request !== undefined){
+                if (request.fulfilled === true){
+                    return request;
+                }
+            }
+        });
+        unfulfilledReq.sort((a, b)=> {
+            return (a!.dateCreated!.valueOf() - b!.dateCreated!.valueOf()); 
+        });
+        fulfilledRequests.sort((a, b)=> {
+            return (a!.dateCreated!.valueOf() - b!.dateCreated!.valueOf());
+        });
+    
+        const sortedRequests : Request[] = unfulfilledReq!.concat(fulfilledRequests!) as Request[];
+        setRequests(sortedRequests);
+      }, [props.requests]);
+      
+
+    const [mutateDeleteRequest] = useMutation(softDeleteRequest);
+    const [mutateRequest] = useMutation(updateRequest);
+    const onSoftDeleteRequest = (index: number) => {
+        const requestsCopy = requests.slice();
+        const req = {...requestsCopy[index]};
+        requestsCopy.splice(index, 1);
+        const id = req._id;
+        props.onChangeNumRequests!(requestsCopy.length);
+        setRequests(requestsCopy);
+        mutateDeleteRequest({variables: {id: id}});
+    }
+
+    const onFulfilledRequest = (index: number) => {
+        const requestsCopy = requests.slice();
+        const req = {...requestsCopy[index]};
+        if(req.fulfilled === false) { 
+            req.fulfilled = true;
+            requestsCopy.splice(index, 1) 
+            let i = requestsCopy.length - 1; 
+            for(; i > -1; --i) {
+                if(requestsCopy[i].fulfilled === false) break;
+                else if(requestsCopy[i]!.dateCreated!.valueOf() < req!.dateCreated!.valueOf()) break;
+            }
+            requestsCopy.splice(i + 1, 0, req);
+        }
+        else {
+            req.fulfilled = false;
+            requestsCopy.splice(index, 1);
+            let i = 0;
+            for(; i < requestsCopy.length; ++i) {
+                if(requestsCopy[i].fulfilled === true) break;
+                else if(requestsCopy[i]!.dateCreated!.valueOf() > req!.dateCreated!.valueOf()) break;
+            }
+            requestsCopy.splice(i, 0, req);
+        }
+        setRequests(requestsCopy);
+        const id = req._id;
+        const requestId = req.requestId;
+        const fulfilled = req.fulfilled;
+        mutateRequest({variables:{request: {id, requestId, fulfilled}}});
+    }
+
+
+    return (
+        <div className="request-list">
```

Comment:
Change class name of wrapping div to match file name. (i.e. change class name to `client-request-table`)
Also create an scss file for this file